dist/clementine.spec.in

```diff
@@ -112,16 +112,16 @@ rm -f $RPM_BUILD_ROOT/usr/share/icons/ubuntu-mono-{dark,light}/apps/24/clementin
 %{_bindir}/clementine
 %{_bindir}/clementine-tagreader
 %dir %{_datadir}/metainfo/
-%{_datadir}/metainfo/clementine.appdata.xml
-%{_datadir}/applications/clementine.desktop
+%{_datadir}/metainfo/org.clementine_player.Clementine.appdata.xml
```

Comment:
There is some inconsistency in the dist directory. The plist file is using org.clementine-player.clementine while the appdata.xml is using org.clementine_player.Clementine. The clementine website is www.clementine-player.org, so maybe we should use the dash instead of the underscore?

Reply:
Ah, I see. The dash isn't allowed in the dbus specification. That's why the underscore was used.